dev/com.ibm.ws.install.featureUtility_fat/fat/src/com/ibm/ws/install/featureUtility/fat/InstallFeatureTest.java

```diff
@@ -344,7 +344,16 @@ public void testFeatureLocalRepoOverride() throws Exception {
     public void testInvalidFeature() throws Exception {
         final String METHOD_NAME = "testInvalidFeature";
         Log.entering(c, METHOD_NAME);
+        replaceWlpProperties("21.0.0.4");
+        copyFileToMinifiedRoot("etc", "../../publish/propertyFiles/publishRepoOverrideProps/featureUtility.properties");
+
+        copyFileToMinifiedRoot("repo/com/ibm/websphere/appserver/features/features/21.0.0.4",
+                "../../publish/repo/com/ibm/websphere/appserver/features/features/21.0.0.4/features-21.0.0.4.json");
 
+        copyFileToMinifiedRoot("repo/io/openliberty/features/features/21.0.0.4",
+                "../../publish/repo/io/openliberty/features/features/21.0.0.4/features-21.0.0.4.json");
+                
+        writeToProps(minifiedRoot+ "/etc/featureUtility.properties", "featureLocalRepo", minifiedRoot + "/repo/");
```

Comment:
This test was failing due to connection issue with maven central and inability to fetch features.json for 21005. These changes will now fetch features.json 21.0.0.4 which already exists in the FAT.